language/python/develop.md

```diff
@@ -225,7 +225,7 @@ Now, to start our application and to confirm that it is running properly, run th
 $ docker-compose -f docker-compose.dev.yml up --build
 ```
 
-We pass the `--build` flag so Docker will compile our image and then starts the containers.
+We pass the `--build` flag so Docker will compile our image and then start the containers.
```

Comment:
Should be starts and not start

Reply:
I believe this should be "start".

"We pass the [...] flag so Docker will compile [...] and then start [...]."
Docker will do two things, "compile" and "start".

With "starts", it'd sound like "Docker will starts the containers."